blocks/liveby-map/liveby-map.css

```diff
@@ -0,0 +1,6 @@
+.liveby-map-main {
+  overflow: hidden;
+  max-width: var(1150px);
```

Comment:
is this the right width? You may want to use padding instead of a max-width, as the sections all have a max already.

Reply:
Sure, I can do that.  Honestly a 100% width might be sufficient if the encapsulating block is already the right width.